src/main/java/edu/kis/powp/jobs2d/command/CommandGenerator.java

```diff
@@ -0,0 +1,51 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandGenerator implements Job2dDriver, ActionListener {
```

Comment:
Mixing driver with ActionListener may be confusing. It would be better to have a separate unit for action listener.